contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -3185,6 +3187,7 @@ TdsSendTypeNumeric(FmgrInfo *finfo, Datum value, void *vMetaData)
 		decString[precision++] = '0';
 	}
 	decString[precision] = '\0';
+	Assert(precision <= outlen);
```

Comment:
wouldn't be `precision == outlen` be better?

Reply:
I will take a look deeper

Reply:
Yes you're right. I will make this change in the next batch of fixes